phpBB/language/en/email/user_remind_username.txt

```diff
@@ -0,0 +1,13 @@
+Subject: {SITENAME} Username reminder
+
+Hello {USERNAME},
+
+Somebody (perhaps you) has requested a reminder of the username for your account at "{SITENAME}". Your Username is: {USERNAME}
```

Comment:
I would say that someone requested a reminder of the username for accounts using the email address XXX personnaly

Reply:
OK